packages/wdio-browserstack-service/src/ai-handler.ts

```diff
@@ -0,0 +1,212 @@
+import aiSDK from '@browserstack/ai-sdk-node'
+import { BStackLogger } from './bstackLogger.js'
+import { TCG_URL, TCG_INFO, SUPPORTED_BROWSERS_FOR_AI } from './constants.js'
+import { handleHealingInstrumentation } from './instrumentation/funnelInstrumentation.js'
+import { createRequire } from 'node:module'
+
+import type { Capabilities } from '@wdio/types'
+import type BrowserStackConfig from './config.js'
+import type { Options } from '@wdio/types'
+import type { BrowserstackHealing } from '@browserstack/ai-sdk-node'
+import path from 'node:path'
+import fs from 'node:fs'
+import url from 'node:url'
+import { getBrowserStackUserAndKey, isBrowserstackInfra } from './util.js'
+import type { BrowserstackOptions } from './types.js'
+
+class AiHandler {
+    authResult: BrowserstackHealing.InitSuccessResponse | BrowserstackHealing.InitErrorResponse
+    wdioBstackVersion: string
+    constructor() {
+        this.authResult = JSON.parse(process.env.TCG_AUTH_RESULT || '{}')
+        this.wdioBstackVersion = createRequire(import.meta.url)('../package.json').version
```

Comment:
add exception handling

Reply:
Exception handling has been added at the appropriate invocations of the methods of this class.